game/board.py

```diff
@@ -0,0 +1,22 @@
+from game.cell import Cell
+from constans.constants_game import (
+    LARGE
+)
```

Comment:
nit: Alphabetical order and one line for single import
```suggestion
from constans.constants_game import LARGE
from game.cell import Cell
```

Reply:
Ok, I will correct all the imports